ng-client/src/app/app.module.ts

```diff
@@ -41,24 +30,17 @@ const extHttpConfig = {
 @NgModule({
   imports: [
     BrowserModule,
-    MaterialModule,
-    HttpModule,
-    FormsModule,
     CoreModule.forRoot(extHttpConfig),
-    EmployeeModule,
-    ProjectModule,
     TimesheetModule,
```

Comment:
I basically just gave up on this for now and included the timesheet module eagerly.